flink-end-to-end-tests/test-scripts/docker-hadoop-secure-cluster/bootstrap.sh

```diff
@@ -92,7 +92,12 @@ if [ "$1" == "--help" -o "$1" == "-h" ]; then
     exit 0
 elif [ "$1" == "master" ]; then
     yes| sudo -E -u hdfs $HADOOP_PREFIX/bin/hdfs namenode -format
-
+    sudo -E sudo netstat -tulpn >> /var/log/hadoop/debugging.out
+    echo "--------------------" >> /var/log/hadoop/debugging.out
+    sudo -E sudo ps -aux >> /var/log/hadoop/debugging.out
+    echo "--------------------" >> /var/log/hadoop/debugging.out
+    sudo -E sudo jps -v >> /var/log/hadoop/debugging.out
```

Comment:
Are you sure that you want to use `sudo -E sudo #...`? I couldn't test it quickly but it looks strange.

Reply:
oh, that's clearly a mistake. Good catch! I'll fix it.